code/modules/surgery/organs/subtypes/xenos.dm

```diff
@@ -116,6 +118,15 @@
 			owner.adjustFireLoss(-heal_amt)
 			owner.adjustOxyLoss(-heal_amt)
 			owner.adjustCloneLoss(-heal_amt)
+		if(IS_HORIZONTAL(owner))
+			processing_ticks++
```

Comment:
This will count up to infinity if they have no viruses. should probably add a length(owner.viruses) check above